packages/react-mutation-mapper/src/component/track/HotspotTrack.tsx

```diff
@@ -85,8 +83,9 @@ export default class HotspotTrack extends React.Component<
             return _.keys(filteredHotspotsByProteinPosStart)
                 .filter(position => Number(position) >= 0)
                 .map(position => ({
-                    codon: Number(position),
+                    startCodon: Number(position),
                     color: '#FF9900',
+                    itemType: TrackItemType.CIRCLE,
```

Comment:
Actually we may have multi-residue cancer hotspots, so it might be better to decide circle vs rectangle somewhere else.